sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/SealedEnumProcessor.kt

```diff
@@ -34,26 +32,20 @@ internal const val ERROR_ELEMENT_IS_ANNOTATED_WITH_REPEATED_TRAVERSAL_ORDER =
 internal const val ERROR_ELEMENT_IS_NOT_KOTLIN_CLASS = "Annotated element is not a Kotlin class"
 internal const val ERROR_ELEMENT_IS_NOT_COMPANION_OBJECT = "Annotated element is not a companion object"
 internal const val ERROR_ENCLOSING_ELEMENT_IS_NOT_KOTLIN_CLASS = "Enclosing element is not a Kotlin class"
```

Comment:
I couldn't think of a scenario with real code that will actually cause this error to be emitted, given the current order of checks (checking for a companion object before). I suppose I could change the order to allow this error to surface...